paasta_tools/contrib/check_orphans.py

```diff
@@ -36,7 +41,7 @@ class ExitCode(Enum):
 
 def get_zk_hosts(path: str) -> List[str]:
     with open(path) as f:
-        x = yaml.load(f)
+        x = yaml.load(f, Loader=SafeLoader)
```

Comment:
can you just do `yaml.safe_load` instead? This is the method we use in other parts of the codebase.